is_prime.py

```diff
@@ -0,0 +1,10 @@
+"""
```

Comment:
small nit, if you are writing docsctrings about a function. You should put it inside the function's indentation. This allows for IDE tools to link the docstring correctly to the function. 

https://peps.python.org/pep-0257/